bundle/DependencyInjection/Compiler/ServiceOverridesCompilerPass.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\DependencyInjection\Compiler;
+
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Netgen\Bundle\SiteBundle\RichText\Converter\Link;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
+
+final class ServiceOverridesCompilerPass implements CompilerPassInterface
```

Comment:
We name compiler passes with specific names, what good is it to have a generic compiler pass when you need to have a different logic for every service within it?

Just name this e.g. `DirectDownloadPass` and work only with `netgen.ibexa_site_api.ezrichtext.converter.link`.

Reply:
fixed in e25dd356ef0faf9fae5d5ad812de1e7b6a4d0645